### PR TITLE
[cyberkongz] removed liquidity pools from voting and supply

### PR DESCRIPTION
**Pull request by frosti-eth**
Changes proposed in this pull request:
- blacklist LP addresses from voting and remove their balances from total supply

### CODE REVIEW

File: src/strategies/cyberkongz/index.ts [hunk 1]
```diff
@@ -28,15 +43,25 @@ export async function strategy(
...
     blockTag
   });
 
+  response.forEach((value: any, i: number) => {
+    const address = calls[i][2][0];
+    if (Math.floor(i / addresses.length) == 2 && blacklistedAddresses.find((add) => add === address)) {
+      nanaSupply -= value;
+    }    
+  });
+
```
Comment (ChaituVR) [inline src/strategies/cyberkongz/index.ts:58]: Hello @frosti-eth we can't update this as it can affect existing space that are using this strategy

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by frosti-eth** [comment]
Hello,

The only space that uses this strategy is our own, and our contract addresses are hardcoded in there. I think we’re good to merge this, but if you like we could submit a _v2 PR? Do you need a wallet signature somewhere?

frosti

**Comment by frosti-eth** [comment]
Did we want to run the 500 address test again? That part of the code was not changed from the existing version

**Comment by ChaituVR** [comment]
@frosti-eth I think it is better to create a -v2 strategy than changing the existing one. so we don't effect existing proposals :)

**Pull request closed without merging**